tests/e2e/login/db-sync-filter.spec.js

```diff
@@ -189,7 +189,7 @@ describe('db-sync-filter', () => {
   };
 
   const getServerRevs = async (docIds) => {
-    const result = utils.requestOnMedicDbNative({ path: '/_all_docs', params: { keys: docIds } });
+    const result = await utils.requestOnMedicDbNative({ path: '/_all_docs', params: { keys: docIds } });
     return result.rows;
```

Comment:
This is a good case to do the await in the function and let the async function wrap the result. In many cases we just want to do something and return the promise directly, but here the function is only intended to return `rows` and not make the caller deal with (or have to know about) any sort of `result` object in the mix. That's good design. Doing the async await like this retains that design. Good stuff!

Reply:
whops!